app/models/chargeback_rate.rb

```diff
@@ -45,26 +45,59 @@ def self.set_assignments(type, cb_rates)
   end
 
   def self.seed
-    fixture_file = File.join(FIXTURE_DIR, "chargeback_rates.yml")
-    if File.exist?(fixture_file)
-      fixture = YAML.load_file(fixture_file)
+    MiqRegion.my_region.lock do
+      # seeding the measure fixture before seed the rates fixtures
+      fixture_file_measure = File.join(FIXTURE_DIR, "chargeback_rates_measures.yml")
+      if File.exist?(fixture_file_measure)
```

Comment:
It might be nice to break this up into 2 smaller methods for reability.  One for seeding the first file, and another for seeding the second file.